site/content/reference/getenvoy_extension_run.md

```diff
@@ -0,0 +1,75 @@
++++
+title = "getenvoy extension run"
+type = "reference"
+parent = "extension"
+command = "run"
++++
+## getenvoy extension run
+
+Run Envoy extension in the example setup.
+
+### Synopsis
+
+
+Run Envoy extension in the example setup.
+
+```
+getenvoy extension run [flags]
+```
+
+### Examples
+
+```
+
+  # Run Envoy extension in the "default" example setup
```

Comment:
Maybe we should link each "default" to actual description of default example and default toolchain?
For someone reading it first time it is hard to know what "default" means.

Reply:
This Markdown is generated automatically out of command definitions in Golang code.

If I add a Markdown link into Golang code, e.g. `["default" example setup](https://getenvoy.io/reference/getenvoy_extension_toolkit_reference/#example-setups)`, it will appear verbatim in the output of `getenvoy extension run --help`, e.g.

```
  # Run Envoy extension in the ["default" example setup](https://getenvoy.io/reference/getenvoy_extension_toolkit_reference/#example-setups) using a particular Envoy release provided by getenvoy.io
  getenvoy extension run --envoy-version wasm:1.15
```

Is it ok ?